src/ui_fsmenu/about.cc

```diff
@@ -56,20 +56,13 @@ FullscreenMenuAbout::FullscreenMenuAbout(FullscreenMenuMain& fsmm)
 		log_err("%s", err.what());
 	}
 
-	graphic_resolution_changed_subscriber_ = Notifications::subscribe<GraphicResolutionChanged>(
```

Comment:
I think this window could benefit from having more height with bigger screen resolutions. We can leave that for later though if you want.